src/plugins/intel_gpu/src/graph/impls/cpu/detection_output.cpp

```diff
@@ -11,16 +11,19 @@
 #include <stdexcept>
 #include <string>
 #include <type_traits>
-#include <immintrin.h>
-#include <xmmintrin.h>
 #include <vector>
 #include <utility>
 
+#ifdef HAVE_SSE
+#include <immintrin.h>
+#include <xmmintrin.h>
+#endif // HAVE_SSE
+
 #ifdef FIX_OPENMP_RELEASE_ISSUE
```

Comment:
@vladimir-paramuzov looks like a dead code.
If threading is really needed there, maybe we need to use TBB?

Reply:
Yeah, this is a legacy code. will be removed in https://github.com/openvinotoolkit/openvino/pull/16014

Reply:
merged with latest chnages